assembl/lib/frontend_urls.py

```diff
@@ -219,8 +244,9 @@ def get_post_url(self, post):
             return self.get_discussion_url() + self.get_relative_post_url(post)
         else:
             route = None
-            phase = get_current_phase_identifier(
-                self.discussion.timeline_events)
+            phase = post.get_created_phase()
+            # The created post must be created within an associated phase
+            assert phase
             first_idea = None
```

Comment:
In this case, remove get_current_phase_identifier completly from the code and tests. Don't keep dead code.

Reply:
The get_current_phase_identifier can have its uses as a utility function, no? It's tracking the time *now* to the timelines. When the post is created is unrelated to the reference of now to timeline dates.

Reply:
Please don't think future hypothetic use. It's not used right now, just remove it. If one day you want the function, just revert the commit.

Reply:
Is get_current_phase_identifier really not used? I see it used below for a route. I'm not sure I understand your comment

Reply:
Please look at the latest diff, it's not used anymore.

Reply:
Oh yes I saw that after, in these cases please don't comment in this commit but in the general diff or general conversation if it is possible